### ISSUE TITLE
Mother's HIV status missing in Health Prevention questionnaire for young infants

### ISSUE THREAD

**Issue by kaledip** [WorldHealthOrganization/smart-emcare-cmr#13]

**Comment by magp18** [comment]
Yes, The form on health prevention was not showing since the plan definition is not working yet. I have implemented it in the Assessment test as a workaround since we can import questionnaires there as a workaround for now, is it okay for you Dipti?
@delcroip

**Comment by kaledip** [comment]
Yes lets go ahead with the workaround because plan definition is still to be reviewed and implemented by L4, o might take more time.

**Comment by magp18** [comment]
OK sure, I have done it like that 👍
